### PR TITLE
Publish the Protect Your API Endpoints guide

### PR DESCRIPTION
**Pull request by nbarbettini**
* Removed nodejs stubs (future task to add Node content)
* Made some wording consistent with other guides
* Added this guide to the TOC
* Tested locally

### CODE REVIEW

File: packages/@okta/vuepress-site/guides/protect-your-api/before-you-begin/springboot/create-app.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
-The [Spring Boot Getting Started Guide](https://spring.io/guides/gs/spring-boot/) teaches you the basics of building a Spring Boot application.
+The [Spring Boot Getting Started Guide](https://spring.io/guides/gs/spring-boot/) teaches you the basics of building a Spring Boot application. You can create a new Spring Boot project by visting the [Spring Initializr](https://start.spring.io) and selecting the dependencies **Web**, **Security**, and **Okta**, and then clicking **Generate Project** to download a zip file.
```
Comment (laura-rodriguez) [inline packages/@okta/vuepress-site/guides/protect-your-api/before-you-begin/springboot/create-app.md]: Is [Spring **Initializr**] correct?

Comment (bdemers) [reply packages/@okta/vuepress-site/guides/protect-your-api/before-you-begin/springboot/create-app.md]: yup :) don't ask me how they came up with the name though

File: packages/@okta/vuepress-site/guides/protect-your-api/configure-packages/springboot/independ.md [hunk 2]
```diff
@@ -1,20 +1,4 @@
...
-  -d artifactId=demo \
-  -d dependencies=web,security,okta \
-  -d language=java \
-  -d type=maven-project \
-  -d baseDir=demo \
-| tar -xzvf -
-```
-
-For existing projects, just include the following Apache Maven dependency: 
+Add the Otka Spring Boot SDK by including the following Apache Maven dependency:
```
Comment (bdemers) [inline packages/@okta/vuepress-site/guides/protect-your-api/configure-packages/springboot/independ.md]: SDK -> Starter

Comment (bdemers) [reply packages/@okta/vuepress-site/guides/protect-your-api/configure-packages/springboot/independ.md]: Spring Boot uses the term "starter".  We also have a "starter" for the management SDK.

**Pull request merged**